openerp/fields.py

```diff
@@ -1504,6 +1624,17 @@ def __init__(self, selection=Default, string=Default, **kwargs):
             selection = api.expected(api.model, selection)
         super(Selection, self).__init__(selection=selection, string=string, **kwargs)
 
+    @property
+    def column_type(self):
+        if self.size == -1 or (self.selection and
```

Comment:
Do we want to support this hack?
Shouldn't we have a better way to tell we want to store integer?